### PR TITLE
Replace SPAlert with NSAlert swift extension

### PR DESCRIPTION
**Pull request by Kaspik**
- replace SPAlert with our Swift version of NSAlert for ARC compatibility

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -1,36 +1,36 @@
 PODS:
-  - Firebase/CoreOnly (7.0.0):
-    - FirebaseCore (= 7.0.0)
-  - Firebase/Crashlytics (7.0.0):
+  - Firebase/CoreOnly (7.1.0):
+    - FirebaseCore (= 7.1.0)
+  - Firebase/Crashlytics (7.1.0):
```
Comment (Jason-Morcos) [inline Podfile.lock:4]: In the future, pod updates really should be committed as separate, atomic PRs. It makes this PR pretty far to review. As long as everyone only runs pod install locally, pods shouldn't be updated all the time. We could just run pod update once as an atomic PR prior to each release.

**Review by Jason-Morcos — APPROVED** [review]
I assume this is a global find and replace and we're gonna test it a lot before release so looks good to merge to the 3.0 branch!

**Comment by Kaspik** [comment]
It's not because the old code didn't have unified structure.. it's couple of painful hours 😃

File: Podfile.lock (hunk 1 above)
Comment (Kaspik) [reply Podfile.lock:4]: It's part of this because it's still not released (this branch) and 7.0.0 had critical bug with crashes. :)

**Pull request merged**